runtime/src/main/java/io/syndesis/integration/runtime/SyndesisRouteBuilder.java

```diff
@@ -281,6 +261,15 @@ protected RouteDefinition fromOrTo(RouteDefinition route, String name, String ur
         return route;
     }
 
+    private String stripPrefix(String trigger, String ... prefixes) {
```

Comment:
Return Optional<String>; even for private method?

Reply:
thats all old  _funktion_ code originally, I just inherited the baggage. 

The logic is a bit complex here, so